examples/websearch/bing_web_search.yaml

```diff
@@ -0,0 +1,33 @@
+# This is a Bing search engine workflow
+# The input payload is:
+# <text>
+#
+# Do not forget to set the bing api_key in the below configuration.
+
+log:
+  stdout_log_level: INFO
+  log_file_level: INFO
+  log_file: solace_ai_connector.log
+
+flows:
+  - name: google_web_search_flow
+    components:
+      # Input from a standard in
+      - component_name: stdin
+        component_module: stdin_input
+
+    # Using Custom component
+      - component_name: web_search_component
+        component_module: websearch_bing
+        component_config:
+          engine: bing
+          api_key: <enter the api key>
+          safesearch: Moderate
+          count: 2
+          detail: false
+        input_selection:
+          source_expression: previous
```

Comment:
I removed the engine data field. Integrating all web search engines was not a good idea because each engine uses different tags and APIs and so we have to implement complex logics to handle engines in a single class.
Also, its readability will be low since the user has to recognize which fields are necessary for which engine.